assets/src/edit-story/elements/image/scalePanel.js

```diff
@@ -46,8 +46,8 @@ const Container = styled.div`
 	width: ${ ( { width } ) => `${ Math.max( width, MIN_WIDTH ) }px` };
 	height: ${ HEIGHT }px;
 
-	background: ${ ( { theme } ) => theme.colors.bg.v7 };
-	border-radius: 4px;
+	background: ${ ( { theme } ) => theme.colors.bg.v0 + 'EE' };
+	border-radius: 100px;
```

Comment:
Does this look ok with a shorter-width panel?

Reply:
Yup